includes/admin/class-wc-stripe-admin.php

```diff
@@ -0,0 +1,71 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+
+/**
+ * Class that handles various admin tasks.
+ *
+ * @since 5.6.0
+ */
+class WC_Stripe_Admin {
+	/**
+	 * Constructor
+	 *
+	 * @since 5.6.0
+	 */
+	public function __construct() {
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_payments_scripts' ] );
+		add_action( 'woocommerce_admin_field_payment_gateways', [ $this, 'wc_stripe_gateway_container' ] );
+	}
+
+	public function register_payments_scripts() {
+		$payment_gateways_script_src_url    = plugins_url( 'build/payment_gateways.js', WC_STRIPE_MAIN_FILE );
+		$payment_gateways_script_asset_path = WC_STRIPE_ABSPATH . 'build/payment_gateways.asset.php';
+		$payment_gateways_script_asset      = file_exists( $payment_gateways_script_asset_path ) ? require_once $payment_gateways_script_asset_path : [ 'dependencies' => [] ];
+
+		wp_register_script(
+			'WC_STRIPE_PAYMENT_GATEWAYS_PAGE',
+			$payment_gateways_script_src_url,
+			$payment_gateways_script_asset['dependencies'],
+			WC_Stripe::get_file_version( 'build/payment_gateways.js' ),
```

Comment:
The `*.asset.php` files inside the `build` directory already contain a `version` attribute - should we use that instead?
Example: https://github.com/woocommerce/woocommerce-gateway-stripe/blob/baa79e5c199a3dd0145437cea849ecca645be1ab/includes/admin/class-wc-stripe-settings-controller.php#L48-L67

Reply:
Hah! Absolutely. This is what happens when I'm borrowing the bulk of code from another plugin and fail to look around in the codebase I'm bringing it to.